packages/qemu.rb

```diff
@@ -60,21 +61,22 @@ class Qemu < Package
   depends_on 'libseccomp' # R
   depends_on 'libssh' # R
   depends_on 'ncurses' # R
+  depends_on 'vte' # R
   depends_on 'zlibpkg' # R
   depends_on 'zstd' # R
 
   def self.patch
     # Avoid linux/usbdevice_fs.h:88:9: error: unknown type name ‘u8’ error
     FileUtils.mkdir_p 'linux'
     FileUtils.cp "#{CREW_PREFIX}/include/linux/usbdevice_fs.h", 'linux/usbdevice_fs.h'
-    system "sed -i 's,^\\\([[:blank:]]*\\\)u8,\\1__u8,g' linux/usbdevice_fs.h"
+    system "sed -i 's,^\\([[:blank:]]*\\)u8,\\1__u8,g' linux/usbdevice_fs.h"
     system "sed -i 's,<linux/usbdevice_fs.h>,\"linux/usbdevice_fs.h\",g' hw/usb/host-libusb.c"
   end
 
   def self.build
     FileUtils.mkdir_p 'build'
     Dir.chdir 'build' do
-      system "../configure #{CREW_OPTIONS.sub(/--target.*/, '')} \
+      system "mold -run ../configure #{CREW_OPTIONS.sub(/--target.*/, '')} \
         --enable-lto"
       system 'make || make -j1'
```

Comment:
No `mold -run make`?

Reply:
Not necessarily needed. The LD flags should in theory make mold get used automatically. I try to use `mold -run` when it looks like that might not be happening.